src/dcdc.cpp

```diff
@@ -106,7 +106,7 @@ int Dcdc::duty_cycle_delta()
     }
     else if (fabs(lvs->current) > ls_current_max    // current above hardware maximum
         || temp_mosfets > 80                        // temperature limits exceeded
-        || (in->voltage < (in->src_voltage_start - in->neg_droop_res * in->current)
+        || (in->voltage < (in->src_voltage_stop - in->neg_droop_res * in->current)
```

Comment:
Yes, makes sense. Probably we didn't realize that issue before as for solar panels it makes not a big difference. But it makes a difference as soon as we use the controller in boost mode discharging a battery.